lib/main/tabs/medicine_tab.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/cupertino.dart';
+import 'package:pet_log/main/widgets/medicine_item.dart';
+
+class MedicineTab extends StatefulWidget {
+  const MedicineTab({Key? key}) : super(key: key);
+
+  @override
+  _MedicineTabState createState() => _MedicineTabState();
+}
+
+class _MedicineTabState extends State<MedicineTab> {
+  @override
+  Widget build(BuildContext context) {
+    return GridView.count(
+      physics: const NeverScrollableScrollPhysics(),
+      crossAxisCount: 2,
+      childAspectRatio: 1.05,
+      children: [
+        MedicineItem(
+          svgPictureAsset: 'lib/assets/medicine_documents.svg',
+          medicineItemText: 'Documents',
+          onTap: () => {},
+        ),
+        MedicineItem(
+            svgPictureAsset: 'lib/assets/medicine_analyzes.svg',
+            medicineItemText: 'Analyzes',
+            onTap: () => {}),
+        MedicineItem(
+            svgPictureAsset: 'lib/assets/medicine_surgeries.svg',
+            medicineItemText: 'Surgeries',
+            onTap: () => {}),
+        MedicineItem(
+            svgPictureAsset: 'lib/assets/medicine_allergies.svg',
+            medicineItemText: 'Allergies',
+            onTap: () => {}),
+        MedicineItem(
+            svgPictureAsset: 'lib/assets/medicine_veterinarian.svg',
+            medicineItemText: 'Veterinarian',
+            onTap: () => {}),
+        MedicineItem(
+            svgPictureAsset: 'lib/assets/medicine_labtests.svg',
+            medicineItemText: 'Lab. tests',
+            onTap: () => {}),
+      ],
```

Comment:
I do think that we can create a [Map<String, String>](https://metanit.com/dart/tutorial/5.3.php) where we can have something like this: 
{
  'lib/assets/medicine_labtests.svg' : 'Lab. test' (or vice versa - 'lib/assets/medicine_labtests.svg' )
}

And then use GridView.builder to build it from map as it will look better and will be easier to read